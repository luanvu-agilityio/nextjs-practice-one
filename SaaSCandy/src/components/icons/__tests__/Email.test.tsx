import { render } from '@testing-library/react';
import EmailIcon from '../Email';

describe('AddressIcon', () => {
  it('matches snapshot', () => {
    const { container } = render(<EmailIcon />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
