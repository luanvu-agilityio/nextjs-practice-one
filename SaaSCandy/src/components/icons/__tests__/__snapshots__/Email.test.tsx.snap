// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`AddressIcon matches snapshot 1`] = `
<svg
  fill="none"
  height="68"
  viewBox="0 0 68 68"
  width="68"
  xmlns="http://www.w3.org/2000/svg"
>
  <rect
    fill="#FFDFAF"
    height="68"
    rx="34"
    width="68"
  />
  <g
    clip-path="url(#clip0_47_5617)"
  >
    <mask
      height="36"
      id="mask0_47_5617"
      maskUnits="userSpaceOnUse"
      width="36"
      x="16"
      y="16"
    >
      <path
        d="M52 16H16V52H52V16Z"
        fill="white"
      />
    </mask>
    <g
      mask="url(#mask0_47_5617)"
    >
      <path
        d="M16.4289 28.8623L18.1797 27.5707V33.3117L16 32.2268V29.711C16 29.3764 16.1589 29.0614 16.4289 28.8623Z"
        fill="#1642B9"
      />
      <path
        d="M52 29.711V32.2268L49.8203 33.3117V27.5707L51.5711 28.8623C51.8411 29.0614 52 29.3764 52 29.711Z"
        fill="#1642B9"
      />
      <path
        d="M52 34.5829V50.9453C52 51.5276 51.5274 52 50.9453 52H17.0547C16.4725 52 16 51.5276 16 50.9453V34.5829C17.0577 35.1097 32.4978 42.7978 33.5296 43.3115C33.678 43.3853 33.839 43.4219 34 43.4219C34.161 43.4219 34.322 43.3853 34.4704 43.3115C35.4738 42.812 51.7636 34.7006 52 34.5829Z"
        fill="#1642B9"
      />
      <path
        d="M37.1641 18.1094H30.8359L33.3672 16.2109C33.742 15.9297 34.2581 15.9297 34.6329 16.2109L37.1641 18.1094Z"
        fill="#1642B9"
      />
      <path
        d="M46.6562 20.2188H21.3438C20.7616 20.2188 20.2891 20.6913 20.2891 21.2734V34.3621L34 41.1894L47.7109 34.3621V21.2734C47.7109 20.6913 47.2385 20.2188 46.6562 20.2188ZM34 32.875H27.6718C27.0897 32.875 26.6171 32.4025 26.6171 31.8203C26.6171 31.2382 27.0897 30.7656 27.6718 30.7656H34C34.5821 30.7656 35.0547 31.2382 35.0547 31.8203C35.0547 32.4025 34.5821 32.875 34 32.875ZM40.3282 28.6562H27.6718C27.0897 28.6562 26.6171 28.1838 26.6171 27.6015C26.6171 27.0194 27.0897 26.5469 27.6718 26.5469H40.3282C40.9103 26.5469 41.3829 27.0194 41.3829 27.6015C41.3829 28.1838 40.9103 28.6562 40.3282 28.6562Z"
        fill="white"
      />
    </g>
  </g>
  <defs>
    <clippath
      id="clip0_47_5617"
    >
      <rect
        fill="white"
        height="36"
        transform="translate(16 16)"
        width="36"
      />
    </clippath>
  </defs>
</svg>
`;
